data.json

```diff
@@ -1,1286 +1,1287 @@
+        "imageUrl": "https://user-images.githubusercontent.com/32130871/117547009-e4586980-b04a-11eb-8daa-baad5ffb4949.png",
+        "issueNumber": 482,
+        "name": "Sulthan Mohaideen"
+    },
+    {
+        "githubUsername": "Surajbokde",
+        "imageUrl": "https://user-images.githubusercontent.com/66197682/105803643-eaf33180-5fc3-11eb-8f97-42de7cee0f05.png",
+        "issueNumber": 279,
+        "name": "Suraj Bokde"
+    },
+    {
+        "githubUsername": "Tarun-Kamboj",
+        "imageUrl": "https://user-images.githubusercontent.com/56023742/112745083-62910d00-8fc3-11eb-9f48-ee1cbc7bacd9.PNG",
+        "issueNumber": 419,
+        "name": "Tarun Kamboj"
+    },
+    {
+        "githubUsername": "TawfikYasser",
+        "imageUrl": "https://user-images.githubusercontent.com/54971231/107149052-42e44d80-695f-11eb-9fbf-83c20f4a3e0a.png",
+        "issueNumber": 227,
+        "name": "Tawfik Yasser"
+    },
+    {
+        "githubUsername": "ThalesBMC",
+        "imageUrl": "https://user-images.githubusercontent.com/52232302/124358775-7c796600-dc2a-11eb-9e2f-e9d9daf27bf9.jpg",
+        "issueNumber": 570,
+        "name": "Thales Brederodes Montarroyos Candido"
+    },
+    {
+        "githubUsername": "Utkarsh1504",
+        "imageUrl": "https://user-images.githubusercontent.com/76392681/107881996-f673b180-6f0c-11eb-90a8-9365ff3e4753.png",
+        "issueNumber": 361,
+        "name": "Utkarsh Mishra"
+    },
+    {
+        "githubUsername": "Utkarsh299-tech",
+        "imageUrl": "https://user-images.githubusercontent.com/60184229/103612077-a7b81c80-4f49-11eb-90eb-0002bc43e97c.png",
+        "issueNumber": 217,
+        "name": "Utkarsh Singh"
+    },
+    {
+        "githubUsername": "UtkarshMishra12",
+        "imageUrl": "(https://user-images.githubusercontent.com/67385503/126064330-eb1f90d9-d7d3-462c-b032-b381ed00153c.png)",
+        "issueNumber": 600,
+        "name": "Utkarsh Mishra"
+    },
+    {
+        "githubUsername": "VasudevJaiswal",
+        "imageUrl": "https://user-images.githubusercontent.com/85981735/129255772-5bac726e-d98e-4eb4-8c9f-c2206b72c976.png",
+        "issueNumber": 641,
+        "name": "Vasudev Jaiswal"
+    },
+    {
+        "githubUsername": "VedantKhairnar",
+        "imageUrl": "https://user-images.githubusercontent.com/624760/88165360-7b177f00-cc0d-11ea-804a-776639dcca29.png",
+        "issueNumber": 18,
+        "name": "Vedant Khairnar"
+    },
+    {
+        "githubUsername": "Yaseen549",
+        "imageUrl": "https://user-images.githubusercontent.com/43935210/115549892-d377cc00-a2c6-11eb-9b4b-7983603b5c6c.jpg",
+        "issueNumber": 449,
+        "name": "Mohamed Yaseen Sheriff S"
+    },
+    {
+        "githubUsername": "YashIndane",
+        "imageUrl": "https://user-images.githubusercontent.com/53041219/99989860-22353e80-2dd9-11eb-95c5-07dd04eecd7d.png",
+        "issueNumber": 180,
+        "name": "Yash Indane"
+    },
+    {
+        "githubUsername": "aakarshteja",
```

Comment:
Why are these not in the first lines? They start with `a` (same in other parts of the file).

Reply:
oh that it is done by names and not by git-hub username